### PR TITLE
libvirt: split up tf files into stages

### PR DESCRIPTION
**Pull request by jstuever**
Split the tf files for libvirt into two stages.
* cluster: Create the permanent infrastructure for the cluster.
* bootstrap: Create the bootstrap infrastructure.

### CODE REVIEW

File: data/data/libvirt/cluster/output.tf [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+output "base_volume_id" {
+  value = libvirt_volume.coreos_base.id
+}
+
+output "network_id" {
+  value = libvirt_network.net.id
+}
+
+output "bootstrap_ip" {
```
Comment (jhixson74) [inline data/data/libvirt/cluster/output.tf:13]: This should probably be output in the bootstrap stage.

**Comment by jhixson74** [comment]
This looks good to me from my limited perspective ;-)

File: data/data/libvirt/cluster/output.tf (hunk 1 above)
Comment (jstuever) [reply data/data/libvirt/cluster/output.tf]: Good idea, and done. Ironically... these IPs are set in the installer, passed into Terraform, and then output back to the installer.

**Comment by jstuever** [comment]
/hold cancel
I sometimes forget master is protected by bugzilla/valid-bug label.

**Pull request merged**